department/requirements.txt

```diff
@@ -0,0 +1,32 @@
+appdirs==1.4.4
```

Comment:
there are too many third-party modules in this file, specify only those you use in the project (at least Django and DRF)